docs/user-guide/usage/node-api.md

```diff
@@ -22,6 +22,10 @@ Stylelint does not bother looking for a `.stylelintrc` file if you use this opti
 
 A string to lint.
 
+### `fix`
+
+Boolean. If `true`, the autofixed code is available as the value of the output property in the returned object.
+
```

Comment:
```suggestion
```

Let's move this to the [options pages](https://github.com/stylelint/stylelint/blob/main/docs/user-guide/usage/options.md#fix) as it's intentionally deduped.

We can add the line somewhere in the middle, so like:

---

CLI flag: `--fix`

Automatically fix, where possible, problems reported by rules.

For CSS with standard syntax, Stylelint uses [postcss-safe-parser](https://github.com/postcss/postcss-safe-parser) to fix syntax errors.

When using the Node.js API, the autofixed code is available as the value of the `output` property in the returned object.

If a source contains a:

- scoped disable comment, e.g. `/* stylelint-disable indentation */`, any problems reported by the scoped rules will not be automatically fixed anywhere in the source
- unscoped disable comment, i.e. `/* stylelint-disable */`, the entirety of source will not be automatically fixed

This limitation in being tracked in [issue #2643](https://github.com/stylelint/stylelint/issues/2643).